actionpack/lib/action_controller/metal/etag_with_flash.rb

```diff
@@ -11,8 +11,10 @@ module EtagWithFlash
 
     include ActionController::ConditionalGet
 
+    delegate :flash, to: :request
```

Comment:
we don't need to add this, in my comments I mentioned this is already defined, that's why we need **request.respond_to?(:flash)** instead of ruby **defined?(flash)**

Reply:
Perfect! Thanks for the help!

Reply:
👍